### PR TITLE
RELATED: SD-652 Update translations

### PR DESCRIPTION
**Pull request by petrjanu**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by naibinh** [comment]
LGTM

**Comment by petrjanu** [comment]
extended test - demo

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4985/ : SUCCESS in 10m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/330/ : SUCCESS in 10s

File: src/internal/translations/pt-BR.json [hunk 1]
```diff
@@ -79,6 +79,14 @@
     "properties.yaxis.title": "Eixo Y",
     "properties.axis.labels": "rótulos",
     "properties.axis.rotation": "Rotação",
+    "properties.axis.name": "nme",
```
Comment (no23reason) [inline src/internal/translations/pt-BR.json]: This is probably a typo?

Comment (no23reason) [reply src/internal/translations/pt-BR.json]: Should be "nome"

Comment (petrjanu) [reply src/internal/translations/pt-BR.json]: okay, THX!

**Comment by yenkins** [comment]
Demo is running at https://client-demo.na.intgdc.com:50032/storybook/

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/331/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2026/ : SUCCESS in 7m 15s

File: src/internal/translations/pt-BR.json (hunk 1 above)
Comment (petrjanu) [reply src/internal/translations/pt-BR.json]: fixed

**Comment by petrjanu** [comment]
extended test - demo

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4986/ : SUCCESS in 6m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/332/ : SUCCESS in 9s

**Comment by yenkins** [comment]
Source code updated. Using staging3.intgdc.com backend now. Demo has already been running at https://client-demo.na.intgdc.com:50032/storybook/

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/333/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2027/ : SUCCESS in 7m 31s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2621/ : SUCCESS in 13m 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4987/ : SUCCESS in 4m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1243/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1641/ : SUCCESS in 18m 06s

**Pull request merged**